site/kubernetes/operator/using-operator.md

```diff
@@ -382,6 +382,10 @@ spec:
       memory: 2Gi
 </pre>
 
+**Note:** It's possible for RabbitMQ to temporary exceed its memory high-water mark which could cause immediate OOM kill.
```

Comment:
It's possible for RabbitMQ to temporarily exceed its memory high-water mark, which could cause an immediate OOM kill.